sys/dev/cp/if_cp.c

```diff
@@ -207,11 +207,11 @@ static struct mbuf *makembuf (void *buf, unsigned len)
 static int cp_probe (device_t dev)
 {
         const struct pci_device_table *cpd;
-	
+
 	cpd = PCI_MATCH(dev, cp_devs);
 	if (cpd == NULL)
 		return ENXIO;
-	device_set_desc(dev, cpd->description);
+	device_set_desc(dev, cpd->descr);
```

Comment:
Same question, though maybe this was in a prior review.

Reply:
Yes, It was converted here.. https://github.com/bsdimp/freebsd/commit/f75f586106003d1671e4f57f951e719b3d6c51af